webapp/src/main/resources/public/js/stores/users/UserStore.js

```diff
@@ -165,6 +150,28 @@ class UserStore {
     onSavePasswordError(err) {
         console.log("error changing password", err);
     }
+
+    getUsers() {
+        this.waitFor(UserSearchParamStore);
+
+        const currPage = 0;
+        const pageSize = 10;
```

Comment:
Nit: I understand why we set the current page to 0 when the search term changes, but it would be nice to have the user's selected page stay the same even if they search again

Reply:
Done